content/posts/2019/08/pycon-sprint-2019/index.eng.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Libplanet at Pycon 2019 Sprint
+date: 2019-08-21
+authors: [dogeon.lee]
+translators: [kidon.seo]
+---
+
+Hello, I'm Lee Dogeon, [Libplanet] committer at Planetarium!
+
+This past August 15th and 16th, our team participated at [Pycon 2019 Sprint] as leader of Project [Libplanet]. In this article, I'd like to briefly talk about the contributions we received at Pycon Sprint.
+
+[Pycon 2019 Sprint]: https://www.pycon.kr/program/sprint?&lang=en-US
+
+## Contributions at Pycon 2019 Sprint
+
+Thanks to all our participants at the Sprint, many features were added to [Libplanet], as well as [Libplanet Explorer] and [Libplanet Explorer Frontend]!
+
+[Libplanet Explorer] is a project that helps you search blocks created in games developed with Libplanet through [GraphQL], and [Libplanet Explorer Frontend] is a TypeScript + React project to make Libplanet Explorer easier to see in your browser.
+
+![balloon](./balloon.jpg)
+
+The Pycon Preparation Committee awarded us a balloon whenever we send a PR (pull request) and so I'd like to share some of the PR's that helped us receive a bunch of these balloons at the event!
```

Comment:
“Organizing Team” seems their choice of name.  See also: <https://www.pycon.kr/contribute/staff?lang=en-US>.

```suggestion
The PyCon Organizing Team awarded us a balloon whenever we send a <abbr title="pull request">PR</abbr> and so I'd like to share some of the PR's that helped us receive a bunch of these balloons at the event!
```